rudder-rest/src/main/scala/com/normation/rudder/rest/lift/SettingsApi.scala

```diff
@@ -535,7 +538,38 @@ class SettingsApi(
     def set = (value : FeatureSwitch, _, _) => configService.set_rudder_featureSwitch_directiveScriptEngine(value)
   }
 
-
+  case object RestOnAcceptPolicyMode extends RestSetting[Option[PolicyMode]] {
```

Comment:
@VinceMacBuche REST API for these two settings

Reply:
Ok I missed it